core/src/main/resources/defaults/clustertemplates/aws/aws-streaming-710.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "CDP 1.3 - Streams Messaging Cluster template",
+  "description": "",
+  "type": "STREAMING",
+  "cloudPlatform": "AWS",
+  "distroXTemplate": {
+    "cluster": {
+      "blueprintName": "CDP 1.3 - Streams Messaging Cluster"
+    },
+    "instanceGroups": [
+      {
+        "name": "master",
+        "template": {
+          "instanceType": "m5.2xlarge",
+          "attachedVolumes": [
+            {
+              "count": 1,
+              "size": 100,
+              "type": "standard"
+            }
+          ],
+          "rootVolume": {
+            "size": 100
+          },
+          "aws": {
+            "encryption": {
+              "type": "NONE"
+            }
+          }
+        },
+        "nodeCount": 1,
+        "type": "GATEWAY",
+        "recoveryMode": "MANUAL"
+      },
+      {
+        "name": "quorum",
+        "template": {
+          "attachedVolumes": [
```

Comment:
Nit: you could use the same attribute order for all instance groups for better readability.